Makefile

```diff
@@ -218,19 +247,19 @@ ifneq "${OSSM_MANIFEST_DATE}" ""
 endif
 
 .PHONY: update-charts
-update-charts: update-1.0-charts update-1.1-charts update-2.0-charts
+update-charts: update-1.0-charts update-1.1-charts update-2.0-charts update-2.1-charts
```

Comment:
Can we drop 1.0 yet?

Reply:
We should do that as part of a separate pr/commit.